sdk/core/azure-core/src/main/java/com/azure/core/util/implementation/SerializableContent.java

```diff
@@ -37,7 +37,7 @@ public SerializableContent(Object content, ObjectSerializer serializer) {
 
     @Override
     public Long getLength() {
-        return null;
+        return (long) toBytes().length;
```

Comment:
NIT: Is returning object Long is better at this place, since the method itself return type is Long. So the autoboxing operation is saved?